sqldelight-compiler/src/test/kotlin/com/squareup/sqldelight/core/queries/InterfaceGeneration.kt

```diff
@@ -571,6 +573,99 @@ class InterfaceGeneration {
+      |  shortArrayValue BLOB AS kotlin.ShortArray NOT NULL
+      |);
+      |
+      |selectAll:
+      |SELECT *, 1
+      |FROM test;
+      |""".trimMargin(), temporaryFolder)
+
+    assertThat(result.errors).isEmpty()
+    val generatedInterface = result.compilerOutput.get(
+        File(result.outputDirectory, "com/example/SelectAll.kt")
+    )
+    assertThat(generatedInterface).isNotNull()
+    assertThat(generatedInterface.toString()).isEqualTo("""
+      |package com.example
+      |
+      |import kotlin.Array
+      |import kotlin.BooleanArray
+      |import kotlin.ByteArray
+      |import kotlin.CharArray
+      |import kotlin.DoubleArray
+      |import kotlin.FloatArray
+      |import kotlin.Int
+      |import kotlin.IntArray
+      |import kotlin.Long
+      |import kotlin.LongArray
+      |import kotlin.ShortArray
+      |import kotlin.String
+      |import kotlin.collections.contentToString
+      |
+      |interface SelectAll {
+      |    val arrayValue: Array<Int>
+      |
+      |    val booleanArrayValue: BooleanArray
+      |
+      |    val byteArrayValue: ByteArray
+      |
+      |    val charArrayValue: CharArray
+      |
+      |    val doubleArrayValue: DoubleArray
+      |
+      |    val floatArrayValue: FloatArray
+      |
+      |    val intArrayValue: IntArray
+      |
+      |    val longArrayValue: LongArray
+      |
+      |    val shortArrayValue: ShortArray
+      |
+      |    val expr: Long
+      |
+      |    data class Impl(
+      |        override val arrayValue: Array<Int>,
+      |        override val booleanArrayValue: BooleanArray,
+      |        override val byteArrayValue: ByteArray,
+      |        override val charArrayValue: CharArray,
+      |        override val doubleArrayValue: DoubleArray,
+      |        override val floatArrayValue: FloatArray,
+      |        override val intArrayValue: IntArray,
+      |        override val longArrayValue: LongArray,
+      |        override val shortArrayValue: ShortArray,
+      |        override val expr: Long
+      |    ) : SelectAll {
+      |        override fun toString(): String = ""${'"'}
+      |        |SelectAll.Impl [
+      |        |  arrayValue: ${'$'}{arrayValue.contentToString()}
+      |        |  booleanArrayValue: ${'$'}{booleanArrayValue.contentToString()}
+      |        |  byteArrayValue: ${'$'}{byteArrayValue.contentToString()}
+      |        |  charArrayValue: ${'$'}{charArrayValue.contentToString()}
+      |        |  doubleArrayValue: ${'$'}{doubleArrayValue.contentToString()}
+      |        |  floatArrayValue: ${'$'}{floatArrayValue.contentToString()}
+      |        |  intArrayValue: ${'$'}{intArrayValue.contentToString()}
+      |        |  longArrayValue: ${'$'}{longArrayValue.contentToString()}
+      |        |  shortArrayValue: ${'$'}{shortArrayValue.contentToString()}
+      |        |  expr: ${'$'}expr
+      |        |]
+      |        ""${'"'}.trimMargin()
```

Comment:
Seeing this triple-quote in triple-quote nonsense makes me really pine after the variable-length delimiters so we could, in theory, use more than 3 for this string so the contained string could still use 3.